src/style.css

```diff
@@ -1 +1,45 @@
 /* add styles here */
+html{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Be consistent with your margins. It seems you are applying the `margin-right` property to all `.stars__star` elements. This could potentially lead to margin collapse. Consider applying margins only to the top or bottom, but not both.